kcbq-connector/src/main/java/com/wepay/kafka/connect/bigquery/SchemaManager.java

```diff
@@ -89,6 +90,7 @@
    *                                    If set to null, ingestion time-based partitioning will be
    *                                    used instead.
    * @param clusteringFieldName
+   * @Param cache The cache containing BigQuery tables that have been queried and retrieved successfully.
```

Comment:
@mubeen can you please revert _**all**_ changes to this file as none appear necessary?

Reply:
Hi @C0urante 

File was reverted back to commit 26ecf5e9f8d3d524cea1dec999d10a222f74539f